.buildkite/pipeline_cross.py

```diff
@@ -21,6 +21,9 @@
     instances_x86_64 = ["c5n.metal", "m5n.metal", "m6i.metal", "m6a.metal"]
     instances_aarch64 = ["m7g.metal"]
     commands = [
+        # we run 0 tests for the side effect of downloading the artifacts. We
```

Comment:
could be worth explaining why this command ends up running 0 tests (I presume it's due to `no_block_pr`)

Reply:
We remove this bti when converting this script to a test in the next commit, so I'd rather just leave it. This is just an interim solution until the next commit.